sites/hashdev/src/_pages/blog/4_announcing-error-stack.mdx

```diff
@@ -136,7 +134,7 @@ Based on our experiences outlined above, we made a few additional design decisio
 
 - _Note: It's still possible to create something like `Error::Unique` but we do not provide shortcut functionality for that within the library, hoping that a lot of its uses can instead be solved through “attachments”._
 
-# What is **`error-stack`?**
+## What is **`error-stack`?**
```

Comment:
```suggestion
## What is **error-stack?**
```
We're getting rid of code-formatted text in headings until we've managed to update the design to handle them better